package.json

```diff
@@ -8,30 +8,22 @@
     "@turf/centroid": "^6.0.2",
     "@turf/nearest-point-on-line": "^6.0.2",
     "@turf/points-within-polygon": "^5.1.5",
-    "make-error": "^1.1.1",
+    "chai": "^4.2.0",
     "turf-point": "^2.0.1",
     "voronoi": "^1.0.0"
   },
   "devDependencies": {
-    "browserify": "^16.2.3",
-    "chai": "^3.5.0",
-    "grunt": "^1.0.4",
-    "grunt-browserify": "^5.3.0",
-    "grunt-contrib-clean": "^0.7.0",
-    "grunt-contrib-watch": "^1.1.0",
-    "gruntify-eslint": "^5.0.0",
-    "minifyify": "^7.2.1",
+    "esm": "^3.2.25",
     "mocha": "^6.1.4"
   },
   "scripts": {
-    "test": "./node_modules/.bin/mocha test --reporter spec --recursive",
-    "compile": "./node_modules/.bin/grunt build"
+    "test": "./node_modules/.bin/mocha --require esm test --reporter spec --recursive"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/PieceMaker/max-inscribed-circle.git"
   },
-  "author": "",
+  "author": "Jonathan Adams",
   "license": "ISC",
```

Comment:
You probably want to license this as MIT, or LGPL, or something like that. ISC is just the default from node when you don't pick one.

Reply:
Done. Thanks!